### PR TITLE
🌐 [i18n-KO] Translated `guides/model_cards.md` to Korean"

### PR DESCRIPTION
**Pull request by SeungAhSon**
# What does this PR do?

Translated the `guides/model_cards.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/huggingface_hub/issues/1626

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)
Team PseudoLab, may you please review this PR? 
@jungnerd, @heuristicwave, @harheem, @cjfghk5697, @seoulsky-field, @nuatmochoi, @boyunJang, @SeungAhSon, @seoyoung-3060, @junejae

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@jungnerd, @heuristicwave May you please review this PR?

### CODE REVIEW

File: docs/source/ko/guides/model-cards.md [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+`huggingface_hub` 라이브러리는 모델 카드를 만들고 공유하고 업데이트하는 파이썬 인터페이스를 제공합니다. 허브의 모델 카드가 무엇인지, 그리고 실제로 어떻게 작동하는지에 대한 자세한 내용은 [전용 설명 페이지](https://huggingface.co/docs/hub/models-cards)를 방문하세요.
+
+<Tip>
+[신규 (베타)! 우리의 실험적인 모델 카드 크리에이터 앱을 사용해 보세요](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool)
+
+</Tip>
+
+## 허브에서 모델 카드 불러오기
+
+Hub에서 기존 카드를 불러오려면 [`ModelCard.load`] 기능을 사용하면 됩니다. 여기서는 [`nateraw/vit-base-beans`](https://huggingface.co/nateraw/vit-base-beans)에서 카드를 불러옵니다.
```
Comment (jungnerd) [inline docs/source/ko/guides/model-cards.md:16]:
```suggestion
Hub에서 기존 카드를 불러오려면 [`ModelCard.load`] 기능을 사용하면 됩니다. 이 문서에서는 [`nateraw/vit-base-beans`](https://huggingface.co/nateraw/vit-base-beans)에서 카드를 불러오겠습니다.
```
`여기`라는 번역이 조금 모호한 것 같아서 다른 번역을 제안드려봅니다!

File: docs/source/ko/guides/model-cards.md [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+
+  - `card.data`: 모델 카드의 메타데이터와 함께 [`ModelCardData`] 인스턴스를 반환합니다. 이 인스턴스에 `.to_dict()`를 호출하여 표현을 사전으로 가져옵니다.
+  - `card.text`: *메타데이터 헤더를 제외*한 카드의 텍스트를 반환합니다.
+  - `card.content`: *메타데이터 헤더를 포함*한 카드의 텍스트 콘텐츠를 반환합니다.
+
+## 모델 카드 만들기
+
+### 텍스트에서 생성
+
+텍스트로부터 모델 카드를 시작하려면, 카드의 텍스트 내용을 초기화 시 `ModelCard`에 전달하면 됩니다.
```
Comment (jungnerd) [inline docs/source/ko/guides/model-cards.md:35]:
```suggestion
텍스트로 모델 카드의 초기 내용을 설정하려면, 카드의 텍스트 내용을 초기화 시 `ModelCard`에 전달하면 됩니다.
```
`initialize`를 다르게 번역해보았습니다!

File: docs/source/ko/guides/model-cards.md [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+# 내 모델 카드
+"""
+
+card = ModelCard(content)
+card.data.to_dict() == {'language': 'en', 'license': 'mit'}  # True
+```
+ 
+이 작업을 수행하는 또 다른 방법은 f-strings를 사용하는 것입니다. 다음 예에서 우리는:
+
+- 우리가 정의한 메타데이터를 YAML로 변환하기 위해 [`ModelCardData.to_yaml`]을 사용하므로 모델 카드에 YAML 블록을 삽입하는 데 사용할 수 있습니다.
```
Comment (jungnerd) [inline docs/source/ko/guides/model-cards.md:53]:
```suggestion
- 모델 카드에 YAML 블록을 삽입할 수 있도록 [`ModelCardData.to_yaml`]을 사용해서 우리가 정의한 메타데이터를 YAML로 변환합니다.
```
문장을 다듬어보았습니다!

File: docs/source/ko/guides/model-cards.md [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+---
+
+# 내 모델 카드
+
+This model created by [@nateraw](https://github.com/nateraw)
+```
+
+### Jinja 템플릿으로부터
+
+Jinja2가 설치되어 있으면, Jinja 템플릿 파일에서 모델 카드를 만들 수 있습니다. 기본적인 예를 살펴보겠습니다:
```
Comment (jungnerd) [inline docs/source/ko/guides/model-cards.md:90]:
```suggestion
`Jinja2`가 설치되어 있으면, jinja 템플릿 파일에서 모델 카드를 만들 수 있습니다. 기본적인 예를 살펴보겠습니다:
```

File: docs/source/ko/guides/model-cards.md [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+
+# jinja 템플릿 정의
+template_text = """
+---
+{{ card_data }}
+---
+
+# MyCoolModel 모델용 모델 카드
+
+이 모델은 이것과 저것을 합니다.
```
Comment (jungnerd) [inline docs/source/ko/guides/model-cards.md:105]:
```suggestion
이 모델은 이런 저런 것들을 합니다.
```

**Review by jungnerd — COMMENTED** [review]
첫 문서 번역 축하드립니다! 🎉
일부 리뷰를 해보았으니 확인 부탁드려요 :D

File: docs/source/ko/guides/model-cards.md [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+```
+---
+language: en
+license: mit
+library_name: keras
+---
+
+# MyCoolModel 모델용 모델 카드
+
+이 모델은 이것과 저것을 합니다.
```
Comment (SeungAhSon) [inline docs/source/ko/guides/model-cards.md:134]:
```suggestion
이 모델은 이런 저런 것들을 합니다.
```

File: docs/source/ko/guides/model-cards.md [hunk 7]
```diff
@@ -0,0 +1,353 @@
...
+```
+---
+language: fr
+license: apache-2.0
+library_name: timm
+---
+
+# MyCoolModel 모델용 모델 카드
+
+이 모델은 이것과 저것을 합니다.
```
Comment (SeungAhSon) [inline docs/source/ko/guides/model-cards.md:159]:
```suggestion
이 모델은 이런 저런 것들을 합니다.
```

File: docs/source/ko/guides/model-cards.md [hunk 8]
```diff
@@ -0,0 +1,353 @@
...
+# 모델 카드 생성 및 공유
+
+`huggingface_hub` 라이브러리는 모델 카드를 만들고 공유하고 업데이트하는 파이썬 인터페이스를 제공합니다. 허브의 모델 카드가 무엇인지, 그리고 실제로 어떻게 작동하는지에 대한 자세한 내용은 [전용 설명 페이지](https://huggingface.co/docs/hub/models-cards)를 방문하세요.
+
+<Tip>
+[신규 (베타)! 우리의 실험적인 모델 카드 크리에이터 앱을 사용해 보세요](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool)
+
+</Tip>
+
+## 허브에서 모델 카드 불러오기
```
Comment (SeungAhSon) [inline docs/source/ko/guides/model-cards.md:14]:
```suggestion
## 허브에서 모델 카드 불러오기[[load-a-model-card-from-the-hub]]
```

File: docs/source/ko/guides/model-cards.md [hunk 9]
```diff
@@ -0,0 +1,353 @@
...
+card = ModelCard.load('nateraw/vit-base-beans')
+```
+
+이 카드에는 접근하거나 활용할 수 있는 몇 가지 유용한 속성이 있습니다:
+
+  - `card.data`: 모델 카드의 메타데이터와 함께 [`ModelCardData`] 인스턴스를 반환합니다. 이 인스턴스에 `.to_dict()`를 호출하여 표현을 사전으로 가져옵니다.
+  - `card.text`: *메타데이터 헤더를 제외*한 카드의 텍스트를 반환합니다.
+  - `card.content`: *메타데이터 헤더를 포함*한 카드의 텍스트 콘텐츠를 반환합니다.
+
+## 모델 카드 만들기
```
Comment (SeungAhSon) [inline docs/source/ko/guides/model-cards.md:31]:
```suggestion
## 모델 카드 만들기[[create-model-cards]]
```

File: docs/source/ko/guides/model-cards.md [hunk 10]
```diff
@@ -0,0 +1,353 @@
...
+
+### 텍스트에서 생성
```
Comment (SeungAhSon) [inline docs/source/ko/guides/model-cards.md:33]:
```suggestion
### 텍스트에서 생성[[from-text]]
```

File: docs/source/ko/guides/model-cards.md [hunk 11]
```diff
@@ -0,0 +1,353 @@
...
+license: mit
+library: timm
+---
+
+# 내 모델 카드
+
+This model created by [@nateraw](https://github.com/nateraw)
+```
+
+### Jinja 템플릿으로부터
```
Comment (SeungAhSon) [inline docs/source/ko/guides/model-cards.md:88]:
```suggestion
### Jinja 템플릿으로부터[[from-a-jinja-template]]
```

File: docs/source/ko/guides/model-cards.md [hunk 12]
```diff
@@ -0,0 +1,353 @@
...
+# MyCoolModel 모델용 모델 카드
+
+이 모델은 이것과 저것을 합니다.
+
+이 모델은 [@nateraw](https://hf.co/nateraw)에 의해 생성되었습니다.
+```
+
+카드 데이터를 업데이트할 때 [`ModelCard.validate`]를 불러와 Hub에 대해 카드가 여전히 유효한지 확인할 수 있습니다. 이렇게 하면 Hugging Face Hub에 설정된 모든 유효성 검사 규칙을 통과할 수 있습니다.
+
+### 기본 템플릿으로부터
```
Comment (SeungAhSon) [inline docs/source/ko/guides/model-cards.md:166]:
```suggestion
### 기본 템플릿으로부터[[from-the-default-template]]
```

File: docs/source/ko/guides/model-cards.md [hunk 13]
```diff
@@ -0,0 +1,353 @@
...
+    model_id='my-cool-model',
+    model_description="this model does this and that",
+    developers="Nate Raw",
+    repo="https://github.com/huggingface/huggingface_hub",
+)
+card.save('my_model_card_2.md')
+print(card)
+```
+
+## 모델 카드 공유하기
```
Comment (SeungAhSon) [inline docs/source/ko/guides/model-cards.md:189]:
```suggestion
## 모델 카드 공유하기[[share-model-cards]]
```

File: docs/source/ko/guides/model-cards.md [hunk 14]
```diff
@@ -0,0 +1,353 @@
...
+
+풀 요청으로 카드를 푸시하고 싶다면 `push_to_hub`를 호출할 때 `create_pr=True`라고 말하면 됩니다.
+
+```python
+card.push_to_hub(repo_id, create_pr=True)
+```
+
+이 명령으로 생성된 결과 PR은 [여기](https://huggingface.co/nateraw/hf-hub-modelcards-pr-test/discussions/3)에서 볼 수 있습니다.
+
+## 메타데이터 업데이트
```
Comment (SeungAhSon) [inline docs/source/ko/guides/model-cards.md:232]:
```suggestion
## 메타데이터 업데이트[[update-metadata]]
```

**Review by SeungAhSon — COMMENTED** [review]
Anchor 추가했습니다!

File: docs/source/ko/guides/model-cards.md [hunk 15]
```diff
@@ -0,0 +1,353 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+
+# 모델 카드 생성 및 공유
+
+`huggingface_hub` 라이브러리는 모델 카드를 만들고 공유하고 업데이트하는 파이썬 인터페이스를 제공합니다. 허브의 모델 카드가 무엇인지, 그리고 실제로 어떻게 작동하는지에 대한 자세한 내용은 [전용 설명 페이지](https://huggingface.co/docs/hub/models-cards)를 방문하세요.
```
Comment (harheem) [inline docs/source/ko/guides/model-cards.md:7]:
```suggestion
`huggingface_hub` 라이브러리는 모델 카드를 생성, 공유, 업데이트할 수 있는 파이썬 인터페이스를 제공합니다. Hub의 모델 카드가 무엇인지, 그리고 실제로 어떻게 작동하는지에 대한 자세한 내용을 확인하려면 [전용 설명 페이지](https://huggingface.co/docs/hub/models-cards)를 방문하세요.
```

Comment (harheem) [reply docs/source/ko/guides/model-cards.md:7]: - 허브 -> Hub로 통일하기
- 가독성을 위해 '~하고 ~하고 ~는' 라는 표현 대신에 쉼표로 끊을 수 있게 바꿔보았어요!
- 내용은 ~ 방문하세요가 비문인 것 같아, '내용을 확인하려면'을 추가해보았습니다.

File: docs/source/ko/guides/model-cards.md [hunk 16]
```diff
@@ -0,0 +1,353 @@
...
+이 모델은 이것과 저것을 합니다.
+
+이 모델은 [@nateraw](https://hf.co/nateraw)에 의해 생성되었습니다.
+```
+
+카드 데이터를 업데이트할 때 [`ModelCard.validate`]를 불러와 Hub에 대해 카드가 여전히 유효한지 확인할 수 있습니다. 이렇게 하면 Hugging Face Hub에 설정된 모든 유효성 검사 규칙을 통과할 수 있습니다.
+
+### 기본 템플릿으로부터
+
+템플릿을 사용하는 대신 [default template](https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/templates/modelcard_template.md)를 사용할 수도 있습니다. 이것은 여러분이 작성하고 싶은 수많은 섹션이 있는 완전한 기능을 갖춘 모델 카드입니다. 내부적으론 [Jinja2](https://jinja.palletsprojects.com/en/3.1.x/) 를 사용하여 템플릿 파일을 작성합니다.
```
Comment (harheem) [inline docs/source/ko/guides/model-cards.md:168]:
```suggestion
자체 템플릿을 사용하는 대신에, 많은 섹션으로 구성된 기능이 풍부한 [기본 템플릿](https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/templates/modelcard_template.md)을 사용할 수도 있습니다. 내부적으론 [Jinja2](https://jinja.palletsprojects.com/en/3.1.x/) 를 사용하여 템플릿 파일을 작성합니다.
```

File: docs/source/ko/guides/model-cards.md [hunk 17]
```diff
@@ -0,0 +1,353 @@
...
+card = ModelCard.from_template(
+    card_data,
+    model_id='my-cool-model',
+    model_description="this model does this and that",
+    developers="Nate Raw",
+    repo="https://github.com/huggingface/huggingface_hub",
+)
+```
+
+마지막으로 이를 허브로 푸시하겠습니다.
```
Comment (harheem) [inline docs/source/ko/guides/model-cards.md:216]:
```suggestion
마지막으로 이를 Hub로 푸시하겠습니다.
```

File: docs/source/ko/guides/model-cards.md [hunk 18]
```diff
@@ -0,0 +1,353 @@
...
+
+```python
+card.push_to_hub(repo_id)
+```
+
+결과 카드는 [여기](https://huggingface.co/nateraw/hf-hub-modelcards-pr-test/blob/main/README.md)에서 확인할 수 있습니다.
+
+풀 요청으로 카드를 푸시하고 싶다면 `push_to_hub`를 호출할 때 `create_pr=True`라고 말하면 됩니다.
```
Comment (harheem) [inline docs/source/ko/guides/model-cards.md:224]:
```suggestion
PR로 카드를 푸시하고 싶다면 `push_to_hub`를 호출할 때 `create_pr=True`라고 말하면 됩니다.
```

Comment (harheem) [reply docs/source/ko/guides/model-cards.md:224]: 230줄에서 PR이라는 표현을 사용하기도 하셨고, PR이라고 표현하는게 좋을 것 같아 제안해봅니다!

File: docs/source/ko/guides/model-cards.md [hunk 19]
```diff
@@ -0,0 +1,353 @@
...
+card.push_to_hub(repo_id, create_pr=True)
+```
+
+이 명령으로 생성된 결과 PR은 [여기](https://huggingface.co/nateraw/hf-hub-modelcards-pr-test/discussions/3)에서 볼 수 있습니다.
+
+## 메타데이터 업데이트
+
+이 섹션에서는 레포 카드에 있는 메타데이터와 업데이트 방법을 확인합니다.
+
+`메타데이터`는 모델, 데이터셋, 스페이스에 대한 높은 수준의 정보를 제공하는 해시맵(또는 키 값) 컨텍스트를 말합니다. 모델의 `pipeline type`, `model_id` 또는 `model_desc` 설명 등의 정보가 포함될 수 있습니다. 자세한 내용은 [모델 카드](https://huggingface.co/docs/hub/model-cards#model-card-metadata), [데이터셋 카드](https://huggingface.co/docs/hub/datasets-cards#dataset-card-metadata) 및 [스페이스 설정](https://huggingface.co/docs/hub/spaces-settings#spaces-settings) 을 참조하십시오. 이제 메타데이터를 업데이트하는 방법에 대한 몇 가지 예를 살펴보겠습니다.
```
Comment (harheem) [inline docs/source/ko/guides/model-cards.md:236]:
```suggestion
`메타데이터`는 모델, 데이터 세트, Spaces에 대한 높은 수준의 정보를 제공하는 해시맵(또는 키 값) 컨텍스트를 말합니다. 모델의 `pipeline type`, `model_id` 또는 `model_desc` 설명 등의 정보가 포함될 수 있습니다. 자세한 내용은 [모델 카드](https://huggingface.co/docs/hub/model-cards#model-card-metadata), [데이터 세트 카드](https://huggingface.co/docs/hub/datasets-cards#dataset-card-metadata) 및 [Spaces 설정](https://huggingface.co/docs/hub/spaces-settings#spaces-settings) 을 참조하십시오. 이제 메타데이터를 업데이트하는 방법에 대한 몇 가지 예를 살펴보겠습니다.
```

Comment (harheem) [reply docs/source/ko/guides/model-cards.md:236]: - glossary에 맞게 dataset -> 데이터 세트로 바꿔보았어요. 
- 고유명사이므로 Spaces로 살려보았습니다.

**Review by harheem — COMMENTED** [review]
승아님의 첫 번역을 축하드립니다 🤩🥳
앞으로의 번역 활동도 계속 응원하겠습니다 🥰

소소하게 몇 가지 코멘트를 남겨보았습니다. 잘 부탁드려요!

File: docs/source/ko/guides/model-cards.md [hunk 20]
```diff
@@ -0,0 +1,353 @@
...
+```
+
+쓰기 권한이 없는 저장소에 일부 변경 사항을 제안하려는 경우가 종종 있습니다. 소유자가 귀하의 제안을 검토하고 병합할 수 있도록 해당 저장소에 PR을 생성하면 됩니다.
+
+```python
+>>> from huggingface_hub import metadata_update
+>>> metadata_update("someone/model", {"pipeline_tag": "text-classification"}, create_pr=True)
+```
+
+## 평가 결과 포함하기
```
Comment (SeungAhSon) [inline docs/source/ko/guides/model-cards.md:262]:
```suggestion
## 평가 결과 포함하기[[include-evaluation-results]]
```

File: docs/source/ko/guides/model-cards.md [hunk 21]
```diff
@@ -0,0 +1,353 @@
...
+    developers="Nate Raw",
+    repo="https://github.com/huggingface/huggingface_hub",
+)
+card.save('my_model_card_2.md')
+print(card)
+```
+
+## 모델 카드 공유하기[[share-model-cards]]
+
+Hugging Face Hub로 인증받은 경우(`huggingface-cli login` 또는 [`login`] 사용) 간단히 [`ModelCard.push_to_hub`]를 호출하여 카드를 허브에 푸시할 수 있습니다. 이를 수행하는 방법을 살펴보겠습니다.
```
Comment (SeungAhSon) [inline docs/source/ko/guides/model-cards.md:191]:
```suggestion
Hugging Face Hub로 인증받은 경우(`huggingface-cli login` 또는 [`login`] 사용) 간단히 [`ModelCard.push_to_hub`]를 호출하여 카드를 Hub에 푸시할 수 있습니다. 이를 수행하는 방법을 살펴보겠습니다.
```
허브 -> Hub로 통일하기

**Comment by heuristicwave** [comment]
@Wauplin I've reviewed this translation, please merge it.
I have a question, Is there a way to make sure my review commit is reflected in Live Preview?

**Comment by jungnerd** [comment]
> @Wauplin I've reviewed this translation, please merge it. I have a question, Is there a way to make sure my review commit is reflected in Live Preview?

@Wauplin This PR is ready to merge! 
By the way, @heuristicwave is Jihun Lim, co-leader of this team.🤗

**Comment by Wauplin** [comment]
Great! Thanks for the ping! :)
Sorry @heuristicwave I missed the ping 4 days ago but for sure I can merge! :tada: 

Thanks @SeungAhSon @heuristicwave @jungnerd!

**Review by Wauplin — APPROVED** [review]

**Comment by Wauplin** [comment]
> I have a question, Is there a way to make sure my review commit is reflected in Live Preview?

Yes and no. The problem is that I need to click on "Approve and Run" to run the CI which makes the live preview active again. Otherwise the CI won't run. Problem is that I need to click on it after every commit to make the live preview up to date. Given the number of PRs and commits I kinda skipped it but if you want I can go back to it let's say once per day.

**Pull request merged**